sdk/cosmos/azure-cosmos/azure/cosmos/aio/_container.py

```diff
@@ -623,78 +659,54 @@ def query_items_change_feed( # pylint: disable=unused-argument
         :keyword Dict[str, Any] feed_range: The feed range that is used to define the scope.
         :keyword partition_key: The partition key that is used to define the scope
             (logical partition or a subset of a container)
-        :type partition_key: Union[str, int, float, bool, Sequence[Union[str, int, float, bool, None]]]
+        :paramtype partition_key: Union[str, int, float, bool, Sequence[Union[str, int, float, bool, None]]]
         :keyword int max_item_count: Max number of items to be returned in the enumeration operation.
         :keyword start_time: The start time to start processing chang feed items.
             Beginning: Processing the change feed items from the beginning of the change feed.
             Now: Processing change feed from the current time, so only events for all future changes will be retrieved.
             ~datetime.datetime: processing change feed from a point of time. Provided value will be converted to UTC.
             By default, it is start from current ("Now")
-        :type start_time: Union[~datetime.datetime, Literal["Now", "Beginning"]]
-        :keyword Literal["High", "Low"] priority: Priority based execution allows users to set a priority for each
+        :paramtype start_time: Union[~datetime.datetime, Literal["Now", "Beginning"]]
+        :keyword priority: Priority based execution allows users to set a priority for each
             request. Once the user has reached their provisioned throughput, low priority requests are throttled
             before high priority requests start getting throttled. Feature must first be enabled at the account level.
-        :keyword Callable response_hook: A callable invoked with the response metadata.
+        :paramtype priority: Optional[Literal["High", "Low"]]
```

Comment:
I think mode should be mutually exclusive with continuation, correct? Because the ocntinuation determines the mode as well as start from already?